validation_pipeline.ipynb

```diff
@@ -25,12 +25,14 @@
         "DEBUG = False #@param {type:\"boolean\"}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    mean_iso = means_isoscape
</pre><p>What's the point of this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Ack, updated.